com.musala.xml.parser.executable/src/com/musala/xml/parser/executable/Main.java

```diff
@@ -0,0 +1,257 @@
+package com.musala.xml.parser.executable;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map.Entry;
+import java.util.TreeMap;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+
+import com.musala.xml.model.executable.School;
+import com.musala.xml.model.executable.Student;
+import com.musala.xml.model.executable.Teacher;
+
+public class Main {
```

Comment:
General styling and other comments for the other Main,java class aply for the executable/Main.java class as well (for their shared code base.)